extensions/ulid/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ulid",
+  "title": "ULID",
+  "description": "some basic ulid tools",
+  "categories": [
+    "Developer Tools"
+  ],
+  "icon": "logo_512.png",
+  "author": "entwine7250",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generator",
+      "title": "Generate ULID by now",
```

Comment:
Maybe change this command to "Generate ULID based on Exact Current Time". Note that the command's title needs to be "Title Cased"